Session_3/closure.js

```diff
@@ -0,0 +1,14 @@
+//Closures and Scopes
+// http://www.codewars.com/kata/closures-and-scopes/train/javascript
+
+function createFunctions(n) {
+    var callbacks = [];
+
+    for (let i = 0; i < n; i++) {
+        callbacks.push(function() {
```

Comment:
why not use es 6 ?

Just checking if es6 is done or not

Reply:
es6 was not covered when I did this exercise. I will refactor this to es6